src/main/java/SessionOne.java

```diff
@@ -75,5 +81,36 @@ public static void isAPerfectNumber(int number) {
         else
             System.out.println("The entered number " + number + " is not a perfect number");
     }
+
+    public static void nextPrimeNumber(int number){
```

Comment:
Yes, I've just closed like all the things related to CSS that I had open in my browser, and also the things that I was looking for Python. I think you're right is better to commit with one thing at a time, so I will be focusing in Java and Git by now, however, the thing is that I went to Medellin, and I asked them to invite me to the meetings of their study group, and I attended to one of those, and now I have an assignment for it. So, I'm planning to complete that one and leave the group, because I've already committed with it. But, I think in the future it won't make things easy, so just this thing and after that, I'm out.

The other thing, regarding this comment is that at the beginning of this PR, you have told me that I can leave that additional function, that doesn't belong here to avoid get the things more complicated. So, I updated that function just to fix a bug... but I'm totally aware about  that each branch should contain only things related to the specific purpose... so, do you want me to create a new branch and remove this from here? or can we make an exception just for this time?

Reply:
Make the choices you think will better help you to move forward. Even when you think you are only studying Git and Java, you might not realise that you are also, learning: code reviewing, code standards, methodologies, collaborative work, exercising your writing and reading skills in a foreign language, learning mental models and thought structures, think about it. 🤯🧠

Ok, fair enough regarding the function, so it's better to try and wrap up this PR soon. In the future think on features, or milestones, those might help you better plan these sessions and make the best out of them. I will say it again, a 'scrum' board for this project can help you with it.

Reply:
I open to suggestions as well, I just want to make the best, which I think is Java, because is the thing the most I know, if I become an expert on it, it can be better that know little things of many things, what do you think? Also, yes this world of code reviewing, code standards, methodologies, collaborative work, exercising your writing and reading skills in a foreign language, learning mental models and thought structures, is a whole thing! But I'm enjoying it.  :)

In the other hand, sorry about the comment that I did not left, it's not ok assumed that everything there make sense.

Finally I created the [board](https://trello.com/b/UPUyeM5S/java-learning)

Reply:
Ok, by now, let's focus on Java, and filling mental gaps about programming in general. If you have questions about any other topic I will be more than happy to help you. I am glad that you enjoy this, it's really inspiring ❤️. I am modifying the board a bit, hope that you don't mind.

About the comment, it's okay, but in future scenarios will be helpful for your team, and therefore for you as well.

Reply:
Of course, I don't mind. Thanks for doing it.

Reply:
This is what I propose, but would like to hear if it feels good for you as well.

Reply:
Yes, of course!. I have to read a lot, I downloaded some books but there is a lot information there, so I'm bit disoriented, there are so many things to learn... https://drive.google.com/drive/u/2/folders/1DObwFBCSa7aSa_nKNnOnGqBZy3Yr9b6E, I have heard many people telling me that one really important thing is to understand the data structures very well.. so, I think that could be a start point.

I love the new board.

Reply:
That's a good idea data structures are something you will have to handle in any programming language, and even when those can be different between them, the objective is always the same. And yes there's a lot of information around, you already know about typed languages and you are also learning about not typed languages, that's good.

These books are cool, but try not to overcommit with them, this one for example: [clean code](https://drive.google.com/drive/u/0/folders/1DObwFBCSa7aSa_nKNnOnGqBZy3Yr9b6E) It's about good patterns, reading one or two pages a day, or whenever you can and feel like it, might be really insightful, you will encounter with things that you already know, and some others will start to make more sense. Take some notes also if you can.

The Java Specification ones, might be outdated, I think that can be checked online: [Java Language Specification](https://cr.openjdk.java.net/~iris/se/11/latestSpec/java-se-11-jls-draft-diffs.pdf) Still huge books, but when checking about data structures and Idk access modifiers for example, would be a good idea to check them.

That's the bad thing about tech books, some might get outdated pretty quickly 🤷🏻‍♂️

Reply:
Regarding data structures, should I look for that on the internet instead? and yes, I so the tweet that you shared, and I thought exactly the same.

About clean code, that's a good idea, I'm going to start doing it. :)

But this is for 2018, it's still too old?, :/ I don't know if you can have access to some kind of official documentation to avoid this old things. Because sometimes the internet is like to vague, don't you think? or maybe I need to learn how to find better things .. :)

Reply:
Take notes and make questions about the Clean Code book.

That one is the official Java Language Specification, 2018 it's pretty good. Not that all. Official documentation might be too technical, it wouldn't hurt to find something more easy to follow. But finding good resources might be tricky sometimes, it just requires a bit more of time.